tnp/templates/consent/profile.html

```diff
@@ -0,0 +1,395 @@
+		                            
+		                            <h6> 
+		                            	<b>Branch</b>
+		                            </h6>
+		                            {{ education_detail.branch }}
+		                        </div>
+		                    </div>
+		                    <div class="container">
+			                    <div class="row">
+						            <center class="card-content z-depth-2 teal-text text-darken-2">
+					              		Your Details up Here. This is your<strong> CGPA </strong> not your SGPA.
+					              	</center>
+			                    </div>
+		                    </div>
+
+		                    <div class="row">
+		                    	<div class="input-field col s5 m2 offset-s1 offset-m1">
+		                    		
+		                    		<p><b>CGPA Semester 1 </b></p>
+		                    		{{semester_var.1}}
+		                    	</div>
+		                    	<div class="input-field col s5 m2">
+		                    		
+		                    		<p><b> CGPA Semester 2 </b></p>
+		                    		{{semester_var.2}}
+		                    	</div>
+		                    	<div class="input-field col s5 m2 offset-s1">
+		                    		
+		                    		<p><b> CGPA Semester 3 </b></p>
+		                    		{{semester_var.3}}
+		                    	</div>
+		                    	<div class="input-field col s5 m2">
+		                    		
+		                    		<p><b> CGPA Semester 4 </b></p>
+		                    		{{semester_var.4}}
+		                    	</div>
+		                    	<div class="input-field col s5 m2 offset-s1">
+		                    		
+		                    		<p><b> CGPA Semester 5 </b></p>
+		                    		{{semester_var.5}}
+		                    	</div>
+		                    </div>
+
+		                    <div class="row">
+		                    	<div class="input-field col s5 m2 offset-s1 offset-m1">
+		                    		
+		                    		<p><b> CGPA Semester 6 </b></p>
+		                    		{{semester_var.6}}
+		                    	</div>
+		                    	<div class="input-field col s5 m2">
+		                    		
+		                    		<p><b> CGPA Semester 7 </b></p>
+		                    		{{semester_var.7}}
+		                    	</div>
+		                    	<div class="input-field col s5 m2 offset-s1">
+		                    		
+		                    		<p><b> CGPA Semester 8 </b></p>
+		                    		{{semester_var.8}}
+		                    	</div>
+		                    	<div class="input-field col s5 m2">
+		                    		
+		                    		<p><b> CGPA Semester 9 </b></p>
+		                    		{{semester_var.9}}
+		                    	</div>
+		                    	<div class="input-field col s5 m2 offset-s1">
+		                    		
+		                    		<p><b> CGPA Semester 10 </b></p>
+		                    		{{semester_var.10}}
```

Comment:
Do not show headers like `CGPA Semester X` if CGPA for semester X not available. The blank space below it doesn't look good.